care/emr/management/commands/load_specimen_definition.py

```
@@ -0,0 +1,352 @@
+            )
+            specimen.save()
+            logger.debug("Created specimen: %s", data["title"])
+            return specimen
+
+        except Exception as e:
+            error_message = (
+                f"Failed to create specimen '{data.get('title', 'Unknown')}': {e}"
+            )
+            raise RuntimeError(error_message) from e
+
+    def handle(self, *args, **options):
```

Ruff (0.14.10)

Unused method argument: `args`

(ARG002)